plutus-tutorial/markdown/2019-04-11-Gist_integration.md

```diff
@@ -0,0 +1,40 @@
+# Gist Support
+
+In the [Plutus Playground](https://prod.playground.plutus.iohkdev.io/) you can now save your contracts and simulations to Github [gists](https://help.github.com/en/articles/about-gists) and read them from there. 

```

Comment:
I'm not wild about having changelog-linked documentation. This isn't part of our main documentation, so we're more likely to not update it, and then we either have an old out-of-date piece of documentation that people can stumble upon, or we spend time updating these mini-tutorials which people probably won't look at.

I think all I'm really suggesting is that these should be promoted to being general documentation articles for the features and not changelog entries. Then the changelog can be short and link to the docs which is good.